pages/en/lb4/Sequence.md

```diff
@@ -23,10 +47,10 @@ instances handle requests and responses. The `DefaultSequence` looks like this:
 -->
 ```js
 class DefaultSequence {
-  async handle(request: ParsedRequest, response: ServerResponse) {
+  async handle(request, response) {
```

Comment:
any reason for removing the types here?

Reply:
This is weird, the diffs are switched; what's in red is what i introduced and what's in green is stuff that I changed. Originally, I accidentally pushed a commit onto the master branch, so I ended up reverting that and then introduced this PR. I'll see if I can fix this.